### ISSUE TITLE
Where can I find the yaml file that achieves the reported accuracy?

### ISSUE THREAD

**Issue by ZYX-MLer** [Deci-AI/super-gradients#1842]
### 💡 Your Question

Why is there only coco2017_yolo_nas_s in the recipes folder without other versions like m, l, n; and coco2017_yolo_nas_s cannot achieve the accuracy of pretrained models? Where can I find the yaml file that achieves the reported accuracy?

**Comment by shaydeci** [comment]
The original training recipe that we used in order to achieve the reported mAP includes some features were introduced by Deci's research team, and it hasn't been decided for some of them yet whether to add them to the library.
A paper is scheduled to be published in the future which will detail the complete training procedure of YoloNAS on COCO though.
